Variant-Alt tweaks

NoSloppy opened the pull request.

profezzorn commented on style_editor.html:
```diff
@@ -99,15 +99,19 @@
   white-space: nowrap;
+  flex-direction: row;
+  flex-wrap: nowrap;
+  transform: scale(.8);
```
Whut?

profezzorn commented on style_editor.html:
```diff
@@ -99,15 +99,19 @@
+  flex-direction: row;
+  flex-wrap: nowrap;
+  transform: scale(.8);
+  margin-left: -35px;
```
Wat?

profezzorn reviewed (commented):
I think you might have to explain what the CSS is doing here.
Perferably in commments....

NoSloppy commented:
> I think you might have to explain what the CSS is doing here. Preferably in comments....

Fixed. Turns out I had Chrome "workspaces" enabled and it was writing to my local file as I changed and played with CSS in the Dev Tools inspector!
Not great.
